api/src/pcapi/sandboxes/scripts/getters/pro_01_common.py

```diff
@@ -0,0 +1,8 @@
+from pcapi.core.users import factories as users_factories
+from pcapi.sandboxes.scripts.utils.helpers import get_pro_user_helper
+
+
+def create_user_new_nav() -> dict:
```

Comment:
(Suggestion) : Vu que tous les utilisateurs devraient avoir la new nav, je propose de renommer ça
```suggestion
def create_pro_user() -> dict:
```

Reply:
ok